bedrock/base/templates/includes/protocol/navigation/pencil-banner.html

```diff
@@ -0,0 +1,17 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% block page_css %}
+  {{ css_bundle('m24-pencil-banner') }}
+{% endblock %}
```

Comment:
**Issue:** this is a pretty tiny page element, so rather than it needing its own CSS bundle suggest adding this to the `m24/base-navigation-and-footer.scss` file as an import instead?

Reply:
good suggestion!